firebase-abt/src/main/java/com/google/firebase/abt/FirebaseABTesting.java

```diff
@@ -140,6 +140,52 @@ public void removeAllExperiments() throws AbtException {
     removeExperiments(getAllExperimentsInAnalytics());
   }
 
+  /**
+   * Adds an experiment to be active in GA by setting a null triggering condition on the provided
```

Comment:
Maybe 'notes that an experiment is active in experiment metrics reporting'
the call should 'trigger' the experiment regardless of the triggering condition, right? (Anything hitting this wants to manually set the experiment as triggered/active'

Reply:
slightly improved text to reflect this more clearly